src/plugins/files_management/public/components/file_flyout.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiDescriptionList,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHealth,
+} from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import type { FileJSON } from '@kbn/files-plugin/common';
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import { i18nTexts } from '../i18n_texts';
+
+interface Props {
+  file: FileJSON;
+  onClose: () => void;
+}
+
+export const FileFlyout: FunctionComponent<Props> = ({ onClose, file }) => {
+  return (
+    <EuiFlyout ownFocus onClose={onClose} size="m">
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="s">
+          <h2>{file.name}</h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiDescriptionList
+              type="column"
+              listItems={[
+                {
+                  title: i18nTexts.filesFlyoutStatus,
+                  description: (
+                    <EuiHealth
+                      color={
+                        file.status === 'READY'
```

Comment:
What does the READY status mean?

Reply:
It means the file is available for download. I'll map these values to something more human friendly :)

Reply:
Is "Ready to download" too long?  Also, can the status use sentence case instead of all caps?